pandas/io/json.py

```diff
@@ -884,3 +940,175 @@ def _recursive_extract(data, path, seen_meta, level=0):
         result[k] = np.array(v).repeat(lengths)
 
     return result
+
+
```

Comment:
There's no `io/json` directory. Just `io/json.py`. Want me to refactor to `io/json/json.py` and `io/json/schema.py` or just make a `io/json_schema.py`?

And maybe something other than `json.py` so we don't clash w/ the stdlib, unless that's intended.